docs/source/releases/v1_10_0a5.rst

```diff
@@ -138,26 +138,46 @@ Consolidate both dynamic and static sectors under geoips/plugins/yaml/sectors.
 
 ::
 
-  renamed: geoips/plugins/yaml/sectors_dynamic/tc_1024x1024/tc_1km_1024x1024.yaml -> geoips/plugins/yaml/sectors/dynamic/tc_1024x1024/tc_1km_1024x1024.yaml
```

Comment:
This seems fine, but not ideal? What do others think? At least it's <120 chars now.

Reply:
I wish that the trailing lines were indented or something. I suppose that another option might be to migrate to using lists here and it would likely automatically create the indentation.